components/feedstock/stores.js

```diff
@@ -126,7 +139,8 @@ const Store = ({ dataset, color }) => {
               mt: 3,
               mb: 2,
               cursor: dataset.public ? 'pointer' : 'not-allowed', // Changes cursor to indicate disabled state
-              //opacity: dataset.public ? 1 : 0.5, // Reduces opacity to indicate disabled state
+              opacity: dataset.public ? 1 : 0.5, // Grey out the button when disabled
```

Comment:
Actually greying out the button might be more impactful than lowering the opacity of a bright color.

```suggestion
              color: dataset.public ? color : 'secondary', // Grey out the button when disabled
```